packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -64,23 +64,25 @@
   command: vdsm-tool configure --force
   changed_when: True
 
-- name: add vdsm-mom allow rule to fapolicy
-  copy:
-   dest: /etc/fapolicyd/rules.d/32-allow-vdsm-mom.rules
-   group: fapolicyd
-   mode: '0644'
-   remote_src: yes
-   content: |
-     allow perm=any trust=1 : dir=/etc/vdsm/mom.d/ ftype=text/x-lisp
-  when: ansible_distribution == 'RedHat'
-
 - name: collect facts about system services
   service_facts:
 
-- name: restart fapolicy service
-  systemd:
-   state: restarted
-   name: fapolicyd
+- block:
+  - name: add vdsm-mom allow rule to fapolicy
+    copy:
+     dest: /etc/fapolicyd/rules.d/32-allow-vdsm-mom.rules
+     group: fapolicyd
+     mode: '0644'
+     remote_src: yes
+     content: |
+       allow perm=any trust=1 : dir=/etc/vdsm/mom.d/ ftype=text/x-lisp
+
+  - name: restart fapolicy service
+    systemd:
+     state: restarted
+     name: fapolicyd
+
   when:
+    - ansible_distribution == 'RedHat'
```

Comment:
I think you can remove `ansible_distribution == 'RedHat'` since you check if the service is running.
AFAIK fapolicyd < 1.1 doesn't have `/etc/fapolicyd/rules.d/`,perhaps check if the directory exists.

Reply:
added: https://github.com/oVirt/ovirt-engine/pull/256